src/main/java/org/paymentsds/mpesa/internal/MpesaResponse.java

```diff
@@ -0,0 +1,60 @@
+package org.paymentsds.mpesa.internal;
+
+public class MpesaResponse {
```

Comment:
Let's find another name for this, Prefixes don't seem good

Reply:
Thought this would be fine, considering that it's an internal class and won't be exposed to the end developer.

I'm not sure what you have against prefixes :sweat_smile: but:
- if the problem is the Mpesa name, we could rename it to something like `CoreResponse` or `InternalResponse`;
- if it's prefixes in general, we could rename it to `Response`, then we'd use the package name (`internal.Response`) to differentiate it from the public `Response` class.

Reply:
Maybe I've strong opinions about prefixes, but in my opinion we tend use them to mask duplication in our code